.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+python:
+  - "2.7"
+addons:
+  apt:
+    packages:
+      - python-zmq
```

Comment:
Just out of curiosity - is there any particular reason for you were installing ZMQ?